README.md

```diff
@@ -456,10 +456,15 @@ access to HTTP verbs `GET`, and `HEAD`. This also includes any kind of
 QUERY or SEARCH verbs, if supported.
 
 ##### `acl:Write`
-gives access to a class of operations that can modify the resource. In a REST
-API context, this would include `PUT`, `POST`, `DELETE` and `PATCH`. This also
+gives access to a class of operations that can modify an existing resource.
+There are two levels: write permissions on an existing resource allows modifying
+(but not deleting) that resource.
+In a REST API context, this would include `POST` and `PATCH`. This also
```

Comment:
I dislike "REST API context", which should be "HTTP context", but can live with it a the current stage of the document.